task_29/Anton.Panciuc/src/main/java/com/hillel/homework/Student.java

```diff
@@ -0,0 +1,78 @@
+package com.hillel.homework;
+
+import java.io.Externalizable;
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Objects;
+
+public class Student implements Externalizable {
+  private String firstName;
+  private String lastName;
+  private int age;
+  Cesar cesar = new Cesar();
```

Comment:
When you make this class static, you don't need to create it's instances. Think about Math module in the standard library, its methods are also static.